corehq/apps/reports/models.py

```diff
@@ -760,6 +760,9 @@ def send(self):
             self.delete()
             return
 
+        if self.owner.is_deleted():
+            return
```

Comment:
did you consider adding `self.delete()` as per above, and if you did, any reason you chose not to